connector/src/main/java/org.apache.flink/connector/nebula/catalog/NebulaCatalog.java

```diff
@@ -93,11 +160,65 @@ public CatalogDatabase getDatabase(String databaseName) throws DatabaseNotExistE
         }
     }
 
+    /**
+     * @param dataBaseName same as graph space name in nebula graph
+     * @param catalogDatabase catalog implementation
+     * @param ignoreIfExists true if contains [if not exists] clause else false
+     */
+    @Override
+    public void createDatabase(String dataBaseName,
+                               CatalogDatabase catalogDatabase,
+                               boolean ignoreIfExists)
+            throws CatalogException {
+        checkArgument(
+                !isNullOrWhitespaceOnly(dataBaseName), "space name cannot be null or empty.");
+        checkNotNull(catalogDatabase, "space cannot be null.");
+
+        if (ignoreIfExists && listDatabases().contains(dataBaseName)) {
```

Comment:
from the view of product consistency, if `ignoreIfExists` is true, we should also give the positive feedback.

For Nebula, if a space exists, we'll get `SUCCEED` when create a repeated space, even though they have different properties, such as vid_type.
![image](https://user-images.githubusercontent.com/16240361/184277272-192324cc-c69c-4a5d-be89-aa30cbfeab2d.png)

Reply:
sure, there are indeed logical inconsistencies, i'll fix it.

Reply:
According to the [doc](https://docs.nebula-graph.com.cn/3.2.0/3.ngql-guide/9.space-statements/1.create-space/)
![image](https://user-images.githubusercontent.com/42756849/184292759-83d8f0d2-ccc7-4f09-8d35-dafd310c90f0.png)

Now, if the space has already existed and create the space with `IF NOT EXISTS` clause it will do nothing. just as 
[ded8300](https://github.com/vesoft-inc/nebula-flink-connector/pull/66/commits/ded8300039c3cab808b8fad28441ad69e6c0ba97).

Test Result:
```
 INFO [main] - Repeat to create space, db1 already exists, no effect.
```

Do you think it's OK to revise this way?

Reply:
I think that's fine, thanks for this kindly log info.